tests/test_organization.py

```diff
@@ -179,3 +179,47 @@ def test_create_relationship(setUp,
 
     fake_signal.assert_called_with(object_id=composition.id,
                                    sender=Composition)
+
+def is_tab_active(page, tab_name):
+    if 'primary">{}'.format(tab_name) in page.content.decode('utf-8'):
+        return True
+    else:
+        return False
+
+
+@pytest.mark.django_db
+def test_organization_edit_buttons(setUp,
+                                   full_organizations,
+                                   membership_person,
+                                   membership_organization):
+     org = full_organizations[0]
+     composition = Composition.objects.first()
+     person = org.personnel[0]
+     association = org.associations[0]
+     emplacement = org.emplacements[0]
+
+     assert is_tab_active(setUp.get(reverse_lazy('edit-organization', args=[org.uuid])),
+                          'Basics') == True
```

Comment:
The Pythonic way to compare an object to `True` is to use the `is` operator (i.e. `assert is_tab_active(...) is True`) but since we've got an `assert` here I think we can safely drop the comparator altogether:

```suggestion
                          'Basics')
```

Same for the rest of the `assert` statements in this test.